internal/healthcheck/healthcheck.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2021 Google LLC All Rights Reserved.
```

Comment:
I think this package is only used inside `cmd/cloud_sql_proxy`. Can we move this internal package to `cmd/cloud_sql_proxy/internal`?